openwisp_controller/config/static/config/css/change-device-group-form.css

```diff
@@ -0,0 +1,31 @@
+.change-device-group form select {
+  margin: 10px;
+}
+.change-device-group form input[type='submit'] {
+  background: #ba2121;
+  border-radius: 4px;
+  padding: 10px 15px;
+  color: #fff;
+  margin: 10px 0 0 0;
+}
+.change-device-group form input[type='submit']:active,
+.change-device-group form input[type='submit']:focus,
+.change-device-group form input[type='submit']:hover {
+  background: #a41515;
+}
+.change-device-group form .cancel-link {
+  display: inline-block;
+  vertical-align: middle;
+  height: 15px;
+  line-height: 15px;
+  background: #ddd;
+  border-radius: 4px;
+  padding: 10px 15px;
+  color: #333;
+  margin: 10px 0 0 10px;
+}
+.change-device-group form .cancel-link:active,
+.change-device-group form .cancel-link:focus,
+.change-device-group form .cancel-link:hover {
+  background: #ccc;
+}
```

Comment:
Is this CSS copied from somewhere? If yes, can we make a generic CSS to include in multiple places instead of duplicating it?

Reply:
@coveralls yes please, let's follow the DRY principle